docs/examples/gallery/betabinomial.md

```diff
@@ -0,0 +1,97 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+# Beta-Binomial Distribution
+
+<audio controls> <source src="../../_static/betabinomial.mp3" type="audio/mpeg"> This browser cannot play the pronunciation audio file for this distribution. </audio>
+
+The Beta-Binomial distribution is a discrete probability distribution derived from the Binomial distribution, with its probability of success in each trial governed by a Beta distribution. This distribution is characterized by three parameters:  $\alpha$, $\beta$, and $n$, where $\alpha$ and $\beta$ are the shape parameters of the Beta distribution and $n$ is the number of trials in the Binomial distribution.
+
+In Bayesian statistics, the Beta-Binomial distribution is commonly used as a prior for the probability of success in a Bernoulli trial. This makes it particularly useful in modeling scenarios with uncertainty in the success probability.
```

Comment:
```suggestion
```
This sentence applies to the Beta distribution not the BetaBinomial. The support of the BetaBinomial is discrete and {0, ..., n}, thus it can not be used as a prior for a probability.